contracts/linear/src/epoch_actions.rs

```diff
@@ -14,10 +14,14 @@ const MAX_SYNC_BALANCE_DIFF: Balance = 100;
 /// during each epoch.
 #[near_bindgen]
 impl LiquidStakingContract {
-    pub fn epoch_stake(&mut self) -> bool {
+    pub fn epoch_stake(&mut self) -> PromiseOrValue<Option<bool>> {
```

Comment:
This function must return true/false to inform the cronjob whether to call it again or not. Same for epoch_unstake

Reply:
I think now there're 3 execution results: 

1. `true` (execution success)
2. `false` (no need to execute)
3. `null` (execution failed)
 
maybe we can switch `false` and `null`